Styling/README.md

```diff
@@ -281,13 +291,66 @@ Colors store rgba components internally, where each component is in the range `0
 
 For example: `color.red`.
 
+Color components may also be accessed with `.x`, `.y`, `.z`, `.w` and `[0]`, `[1]`, `[2]`, `[3]`.
+
 Colors support the following binary operators by performing component-wise operations: `===`, `==`, `!==`, `!=`, `+`, `-`, `*`, `/`, and `%`.  For example `color() === color()` is true since the red, green, blue, and alpha components are equal.  This is not the same behavior as a JavaScript `Object`, where, for example, reference equality would be used.  Operators are essentially overloaded for `Color`.
 
 Colors have a `toString` function for explicit (and implicit) conversion to strings in the format `'(red, green, blue, alpha)'`.
 * `toString() : String`
 
 Colors do not expose any other functions or a `prototype` object.
 
+#### Vector
+
+The styling language includes 2, 3, and 4 component floating-point vector types: `vec2`, `vec3`, and `vec4`. Vector constructors share the same rules as GLSL:
+
+##### vec2
+
+* `vec2(Number)` - initialize each component with the number
+* `vec2(Number, Number)` - initialize with two numbers
+* `vec2(vec2)` - initialize with another `vec2`
+* `vec2(vec3)` - drops the third component of a `vec3`
+* `vec2(vec4)` - drops the third and fourth component of a `vec4`
+
+##### vec3
+
+* `vec3(Number)` - initialize each component with the number
+* `vec3(Number, Number, Number)` - initialize with three numbers
+* `vec3(vec3)` - initialize with another `vec3`
+* `vec3(vec4)` - drops the fourth component of a `vec4`
+* `vec3(vec2, Number)` - initialize with a `vec2` and number
+* `vec3(Number, vec2)` - initialize with a `vec2` and number
+
+##### vec4
+
+* `vec4(Number)` - initialize each component with the number
+* `vec4(Number, Number, Number, Number)` - initialize with four numbers
+* `vec4(vec4)` - initialize with another `vec4`
+* `vec4(vec2, Number, Number)` - initialize with a `vec2` and two numbers
+* `vec4(Number, vec2, Number)` - initialize with a `vec2` and two numbers
+* `vec4(Number, Number, vec2)` - initialize with a `vec2` and two numbers
+* `vec4(vec3, Number)` - initialize with a `vec3` and number
+* `vec4(Number, vec3)` - initialize with a `vec3` and number
+
+`vec2` components may be accessed with `.x`, `.y` and `[0]`, `[1]`.
```

Comment:
Looks like this is in the **vec4** section.  Either create a new section here or move some of these sentences to be in the right vector type section just above.